docs/static/settings/monitoring-settings.asciidoc

```diff
@@ -44,40 +44,48 @@ shipping monitoring data.
 [[monitoring-collection-settings]]
 ===== Monitoring collection settings
 
-`xpack.monitoring.collection.interval`::
+`monitoring.collection.interval`::
 
 Controls how often data samples are collected and shipped on the Logstash side.
 Defaults to `10s`. If you modify the collection interval, set the 
-`xpack.monitoring.min_interval_seconds` option in `kibana.yml` to the same value.
+`monitoring.min_interval_seconds` option in `kibana.yml` to the same value.
 
+`monitoring.cluster_uuid`::
+
+The universally unique identifier (UUID) for the monitoring cluster. 
+By default, {ls} identifies and uses the `cluster uuid` from the first
+`ouput-elasticsearch` plugin defined in the pipelines.
```

Comment:
I think this <newline> makes the following phrase disconnected from the `monitoring.cluster_uuid` item,  in the rendered docs http://logstash_11714.docs-preview.app.elstc.co/guide/en/logstash/master/monitoring-internal-collection.html#monitoring-collection-settings